workitem/workitemtype.go

```diff
@@ -39,6 +39,13 @@ const (
 	SystemStateInProgress = "in progress"
 	SystemStateResolved   = "resolved"
 	SystemStateClosed     = "closed"
+
+	SystemMarkupPlainText = "PlainText"
```

Comment:
actually, I should remove all but 'markdown' and 'plain text'. The others will be added as we add support for remote work items (JIRA and GitHub). Asciidoctor will come later.